src/Admin/AdminNotices.php

```diff
@@ -249,6 +249,16 @@ protected function render_notices(): void {
 		</style>
 		<?php
 		$count = 0;
+
+		/**
+		 * Fires before the admin notices are rendered.
+		 *
+		 * @param array<mixed> $notices The notices to be rendered
```

Comment:
It's probably overkill to fully-type these, but we can can at least make them `array<string,mixed>` to hint that there's actual string keys that need to be used.